docs/developer-guide/nodejs-api.md

```diff
@@ -241,6 +241,26 @@ Map {
 */
 ```
 
+### Linter#defineParser
+
+Each instance of `Linter` holds a map of custom parsers. If you want to define a parser programmatically you can add this function
+with the name of the parser as first argument and the parser object (having either `parseForESLint` or `parse` as keys) as second argument.
```

Comment:
I think it would be good to link to the parser documentation [here](https://eslint.org/docs/developer-guide/working-with-plugins#working-with-custom-parsers). For example:

```markdown
...with the name of the parser as the first argument and the [parser object](/docs/developer-guide/working-with-plugins#working-with-custom-parsers) as the second argument.
```